cmd/datamon/cmd/cli_test.go

```diff
@@ -156,13 +156,14 @@ func listRepos(t *testing.T) ([]repoListEntry, error) {
 	if err != nil {
 		panic(err)
 	}
-	log.SetOutput(w)
+	logStdOut = func(format string, a ...interface{}) (int, error) {
+		return fmt.Fprintf(w, format, a...)
```

Comment:
```suggestion
		return fmt.Fprintf(w, format + "\n", a...)
```

Reply:
Why not just include it as a the template for CLI?

Reply:
@kerneltime it just looks like you are reinventing the stdlib "log" facility.
The way to go is IMHO:
- create our own instances of `log.Logger`, let's say `outlog` and `errlog`, with `New(os.Stdout, ...)` and `New(os.Stderr, ...)`. These are the one that may be mocked.
- decide how to be consistent with zap loggers (a similar output setting is possible)
- use the loggers with the usual `Fprintf`, `Fatalf` etc,
- do not use `fmt.Fprintf` anywhere
- do not clutter every output with `+"\n"`

Reply:
OK